src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands;
+
+import java.util.Comparator;
+
+import seedu.address.model.person.ReadOnlyPerson;
+
+/**
+ * Sorts the displayed person list.
+ */
+public abstract class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String COMMAND_ALIAS = "sr";
+
+    public static final String MESSAGE_SORTED = "Persons sorted!\n";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the displayed person list.\n"
+            + "Alias: " + COMMAND_ALIAS + "\n"
+            + "Options: \n"
+            + "\tdefault - Sorts first based on whether contact is a favorite, then by name in alphabetical order.\n"
```

Comment:
Would this sound better:
Sorts first by favorite status of a contact, then by name in alphabetical order. 🤔

I did some reading up on the mentions of "_Sorts first by..._":
![sort](https://user-images.githubusercontent.com/2218315/32217691-c4c74636-be63-11e7-9c7b-90c00723cb7a.PNG)